enhancements/console/quick-starts.md

```diff
@@ -176,6 +176,17 @@ The language label must be a two-letter
 The language is also appended to the name of non-English Quick Starts so we can avoid showing
 duplicates of the same Quick Start content in different languages. The format is
 my-quick-start for English and my-quick-start-{language code} for other languages.
+If a Quick Start is not translated into a user's language, the English version will be shown.
+
+Example Explore Serverless Quick Starts CR snippet for Japanese:
+```yaml
+apiVersion: console.openshift.io/v1
+kind: QuickStart
+metadata:
+  name: explore-serverless-{ja}
```

Comment:
Sorry, I meant to use `{lang}` as the replacement variable for `ja` (without the braces). Braces are not allowed in most resource names. This is why the YAML snippet really helps :)

```suggestion
  name: explore-serverless-ja
```